src/Bundle/CoreBundle/Field/Types/TextAreaFieldType.php

```diff
@@ -3,10 +3,63 @@
 namespace UniteCMS\CoreBundle\Field\Types;
 
 use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Validator\ConstraintViolation;
+use UniteCMS\CoreBundle\Entity\FieldableField;
+use UniteCMS\CoreBundle\Field\FieldableFieldSettings;
 use UniteCMS\CoreBundle\Field\FieldType;
 
 class TextAreaFieldType extends FieldType
 {
+
     const TYPE = "textarea";
+
     const FORM_TYPE = TextareaType::class;
+
+    /**
+     * All settings of this field type by key with optional default value.
+     */
+    const SETTINGS = ['rows'];
+
+    function getFormOptions(FieldableField $field): array
+    {
+        return array_merge(
+            parent::getFormOptions($field),
+              [
+                'attr' => [
+                    'rows' => $field->getSettings()->rows ?? 2,
+                    'style' => 'resize: horizontal;'
```

Comment:
Maybe this should be moved to css (https://github.com/unite-cms/unite-cms/tree/master/src/Bundle/CoreBundle/Resources/webpack/sass)